### ISSUE TITLE
Add the ability to select which packages the tokens have access to

### ISSUE THREAD

**Issue by tiagosampaio** [repman-io/repman#329]
The general idea is to give the application the ability to apply a sort of permissions to each token.

The scenarios I imagine:

- A given token can have access to all the packages (global permission)
- A given token can have access only to some specific packages (picked up manually)

Feel free to speak your mind about this idea. It could help us a lot in managing private packages and distribute them to different customers depending on the plan paid or packages purchased.

**Comment by akondas** [comment]
Yes, this is a very good idea that we have thought about before, but in a slightly different version. Nevertheless, this proposal is ok and we will gladly accept it for implementation. At present, I can help in its implementation (mainly in the form of reviews and discussions). If you have any further questions, let me know, I could write down how to start changes :wink:

**Comment by jmalinens** [comment]
In our use case we use gitlab.
It would be cool to use existing gitlab permissions for each user token

**Comment by sunilit42** [comment]
Any update on this?
@tiagosampaio which solution have you used?
